apps/cmp.c

```diff
@@ -2182,7 +2182,16 @@ static int read_config(void)
      * would not make sense within the config file.
      * Moreover, these two options and OPT_VERBOSITY have already been handled.
      */
+    int n_options = sizeof(cmp_options) / sizeof(OPTIONS) - 1;
```

Comment:
How about `OSSL_NELEM()` instead.

Reply:
Thanks for the good hint - done here and also for `cmp_vars` a little further down.